### PR TITLE
Modify olm version in e2e test

### PR DESCRIPTION
**Pull request by varshaprasad96**
**Description of the change:**
Install OLM version "0.15.1" for e2e tests as
install manifests are present in sdk locally.

**Motivation for the change:**
To reduce calls to GitHub.

Issue: #3964 

Note: Latest version of OLM is still being tested in `subcommand-olm-install.sh` and `install_test.go` - this wouldn't eliminate requests to GitHub entirely.

**Checklist**

If the pull request includes user-facing changes, extra documentation is required:
- [ ] Add a new changelog fragment in `changelog/fragments` (see [`changelog/fragments/00-template.yaml`](https://github.com/operator-framework/operator-sdk/tree/master/changelog/fragments/00-template.yaml))
- [ ] Add or update relevant sections of the docs website in [`website/content/en/docs`](https://github.com/operator-framework/operator-sdk/tree/master/website/content/en/docs)

### CODE REVIEW

**Review by jmrodri — APPROVED** [review]
/lgtm

**Review by estroz — APPROVED** [review]
/lgtm

**Review by camilamacedo86 — APPROVED** [review]
/lgtm 

Great work 👍

File: test/e2e-ansible/e2e_ansible_suite_test.go [hunk 1]
```diff
@@ -87,7 +87,7 @@ var _ = BeforeSuite(func(done Done) {
 
 	if isOLMManagedBySuite {
 		By("installing OLM")
-		Expect(tc.InstallOLM()).To(Succeed())
+		Expect(tc.InstallOLMVersion("0.15.1")).To(Succeed())
```
Comment (camilamacedo86) [inline test/e2e-ansible/e2e_ansible_suite_test.go:90]: PS. It could be a conts.

**Pull request merged**